wiki/Help_centre/Upgrading_to_lazer/it.md

```diff
@@ -193,7 +193,7 @@ Lazer prende l'input a 1000Hz indipendentemente dal limitatore di FPS, che è an
 
 Se siete curiosi di sapere come questo influisce sulla latenza in ingresso e di verificare la vostra percezione, eseguite il "Run latency certifier" integrato in fondo alle impostazioni.
 
-You can also [read this technical document](https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) explaining the path we are taking along with rationale behind it.
+Potresti anche [leggere questo documento tecnico](https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) spiegando il percorso che stiamo intraprendendo e le motivazioni dietro a questa decisione.
```

Comment:
I don't think it's clear enough
I propose:
"Potresti anche [leggere questo documento tecnico](https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) in cui viene spiegato il percorso che stiamo intraprendendo e le motivazioni dietro alle decisioni prese."